traitsui/testing/tester/qt4/helpers.py

```diff
@@ -91,6 +91,29 @@ def mouse_click_qwidget(control, delay):
     )
 
 
+def mouse_click_tab_index(tab_widget, index, delay=0):
```

Comment:
Nit:  Since this function is only used internally, we can probably make this `delay` required, this will help us to catch instances where we forget to pass the value.